applications/msai.md

```diff
@@ -0,0 +1,300 @@
+
+- **Project Name:** MSAI Medical Data Chain
+- **Team Name:** Marvsai Team
+- **Payment Address:** 0x8f1013fa606c6fcbcd3eff057e5b320b0c5f72e2
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+## Project Overview
+### Overview
+Many patients use clinics worldwide, but they face difficulty moving patient records, thus they have to take redundant duplicate tests. It is inconvenient and impractical to carry data folders over particularly between countries.  
+To solve this problem we gathered a team and designed an international app. We wanted to store the data in a decentralised manner.
+For further background you can[ read our blog](https://www.notion.so/Why-we-need-a-blockchain-for-health-data-1080c5b727964721822650972c28cab2)
+
+Our goal is to customise a blockchain Pallet/OFW such that a standardized format data storage is made a straight forward part of the ecosystem. The data should be distributable accessible and auditable. Essentially we would like to implement something similar to w3c Solid pod integrated with a Susbtrate node.
+Integrating application specific security and storage for every application is a major pain point which is not addressed by blockchains.
+A combination of a Substrate blockchain and Solid pod api [5] would provide the security, decentralisation and immutable record keeping that is needed to make healthcare interoperable.
+To make this feasible it needs to be split into stages initially using a smart contract developed in  Ink!/ WebAssembly which records ownership and access [1] to a data pod with a similar API to Solid PODs [4].
+
+ID verification and data storage could be integrated through a Sidecar approach ( a containerized system design pattern) allowing other app developers within the ecosystem to reuse the chain without needing their own backend. 
+For a simpler initial alpha we could integrate IPFS as a pod making it simpler than the full Pod api [4].
+Distributed data could be verified as authentic by having a secure hash stored on the chain with the data in the separate pod.
+With trust being established by the blockchain layer, any blockchain node that provides faulty data as verified through smart contract hashes will be kicked out and forfiet their staking in the parachains native token.
+We also propose specific data format for anonymous tags such that no user identified private data would be exposed
+during the whole process. Additional layers of encryption and anonymity would be provided in subsequent stages.
+
+The initial user of this parachain would be our own app where users store any medical documents with automated information extraction, along with authenticated GPDR opt out.
```

Comment:
How do you extract information if the data is encrypted client-side?

Reply:
We extract information with the clients permission when they opt in and use our application. We cannot do this if they use another app.